pkg/storage/wrapper/prefix/prefix_wrapper.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// package prefix offers provider.Store wrappers for stores with certain constraints on ID values such as
+// reserved prefix character values (eg IDs starting with underscore) by prefixing it with IDPrefix.
+
+package prefix
+
+import (
+	"fmt"
+
+	"github.com/hyperledger/aries-framework-go/pkg/storage"
+)
+
+// IDPrefix is the prefix added to IDs in StorePrefixWrapper.
+const IDPrefix = "p"
```

Comment:
With a hard-coded prefix, i have trouble imaging that this functionality is generic outside of a couchdb fix. Normally I would expect a prefix as a solution to multiple key indexes in a single KV database.

Reply:
Sure, I will pass in the prefix to the wrapper instance as an argument, in this single case used here it's for key IDs, so it would be a `k` prefix argument value passed in by the storage consumer.

Reply:
updated